src/Asiago/SlashCommands/IsThereAnyDealModule.cs

```diff
@@ -34,7 +36,7 @@ public IsThereAnyDealModule(IOptions<IsThereAnyDealOptions> itadOptions, HttpCli
                 await ctx.FollowUpAsync(new DiscordFollowupMessageBuilder().AddEmbed(

```

Comment:
make messages from this file consistent with rest of app... don't need embeds for simple messages.

Reply:
I'll do this when I update the ITAD stuff to use the new api.